### DISCUSSION TITLE
Arena launch in the Heroic Dungeons phase

### DISCUSSION THREAD

**Discussion by gabonvaricel** [chromiecraft/chromiecraft#4514] [General]
Bringing arena launch to the heroic dungeons phase will give a lot of content to people faster.
Tier 4 is not hard so the season 1 gear will not break the tier , at least for sure for the normal difficulty of raids. And if the concern is that the gear will make the heroic version  too easy, remove the access into heroic versions with season one gear untill tier 5 launches or we get a realm first.

**Comment by 55Honey** [comment]
Please consider the item level of the potential rewards for S1 Arena and heroics/T4.

Wouldn't this lead to many PVP items being bis for raids once they launch?

**Reply by gabonvaricel** [reply]
The item level of the season one gear is higher than the heroic dungeons level, that is true. But a lot of that is budgetet into higher stamina and resillience, which is useless for pve. Im not sure for dpser and healers it will be better. Maybe for tanks. But another solution is to add arena and not introduce arena points yet or rating .So its just for fun

**Reply by 55Honey** [reply]
Arena skirmish is already there. You can sign up from anywhere, and just type  `.lla queue`

**Reply by gabonvaricel** [reply]
Yeah but few people do it because how unbalanced pvp is without resillience. So when the 70 bracket comes and we get the level 70 blue pvp set which finally has resilliece , but about adding a preseason 0? Where everyone who reaches 2k personal rating gets the tranquillien tabard and a title from cata lets say?

**Reply by 55Honey** [reply]
That's all custom implementations. We can consider such things when our issue counter is near zero.

**Comment by balleny** [comment]
Alternatively, less blizzlike tho: we could make the blue pvp gear accessible via Arena only, until t4 content, and have some kind of season 0 during the heroic dungeon phase.
this could also help ironing out potential bugs which would otherwise affect season 1
